samples/routing/data-distribution/sample.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Data distribution
+summary: Implementing data distribution on top of NServiceBus
+component: Core
+tags:
+- Routing
+- Data distribution
+---
+
+The sample demonstrates how NServiceBus routing can be used to implement data distribution scenarios. Data distribution is about delivering a given piece of information (message) to all instances of the subscribed endpoint, as opposed to a single instance when using the Publish/Subscribe pattern built into NServiceBus. 
+
+## Prerequisites
+
+Make sure MSMQ is set up as described [here](/nservicebus/msmq/). 
+
+## Running the project
+
+ 1. Start all the projects by hitting F5.
+ 1. The text `Press <enter> to send a message` should be displayed in the Client.1 and Client.2 console windows. 
+ 1. Press `<enter>` a couple of times in both Client windows.
+
+
+### Verifying that the sample works correctly
+
+ 1. Notice that for each order message sent from any client there is a single confirmation `Order ABCD accepted` displayed in either Client.1 or Client.2 window. The `OrderAccepted` event is always processed by a single instance of the Client endpoint.
```

Comment:
I think they intro already says that when using pub sub, events are processed by a single instance.